routes/api.php

```diff
@@ -1,8 +1,6 @@
 <?php
 
-use Illuminate\Http\Request;
+use App\Http\Controllers\Api\CityController;
 use Illuminate\Support\Facades\Route;
 
-Route::get('/user', function (Request $request) {
-    return $request->user();
-})->middleware('auth:sanctum');
+Route::apiResource('cities', CityController::class);
```

Comment:
Great use of `Route::apiResource` to quickly define the RESTful routes for the controller. However, in our team, we prefer to explicitly define each route (get, post, patch, delete) to maintain greater control and flexibility in managing specific permissions and middlewares for each operation. For now, this is fine, but keep it in mind! 🚀